packages/nextjs/src/server/errors.ts

```diff
@@ -36,7 +36,7 @@ export const authAuthHeaderMissing = (helperName = 'auth') =>
 - Your Middleware matcher is configured to match this route or page.
 - If you are using the src directory, make sure the Middleware file is inside of it.
 
-For more details, see https://clerk.com/docs/quickstarts/get-started-with-nextjs.
+For more details, see https://clerk.com/docs/quickstarts/nextjs.
```

Comment:
Perfect!

Reply:
I removed the `.` at the end of line because it was considered part of the link and the `.../nextjs.` url does not exist.